docs/project/setup.sh

```diff
@@ -26,6 +26,8 @@ echo setup bpmn-moddle
 cd $base/bpmn-moddle
 npm install
 
+npm run build
```

Comment:
Could also be fixed by changing from `prepublishOnly` to `prepare` [here](https://github.com/bpmn-io/bpmn-moddle/blob/master/package.json#L13).

Reply:
(That would make `bpmn-moddle` `npm link` friendly).

Reply:
true, we also discused this in a weekly. Will follow-up